config/base/derive/src/lib.rs

```diff
@@ -694,41 +728,28 @@ fn gen_impl_default(original: &ViewInput, view: &ViewInput) -> proc_macro2::Toke
     let ViewInput {
         generics,
         ident: view_ident,
-        fields,
         ..
     } = view;
     let (impl_generics, ty_generics, where_clause) = generics.split_for_impl();
-    let field_idents = extract_field_idents(fields);
 
     quote! {
         impl #impl_generics Default for #view_ident #ty_generics #where_clause {
```

Comment:
Consider using fully quilified paths like `std::default::Default`. This is important for macros, cause with just `Default` user can cheat with `use my_default::Default`